os/net/lwip/src/api/sockets.c

```diff
@@ -575,6 +575,12 @@ int lwip_accept(int s, struct sockaddr *addr, socklen_t *addrlen)
 	LWIP_ASSERT("invalid socket index", (newsock >= LWIP_SOCKET_OFFSET) && (newsock < NUM_SOCKETS + LWIP_SOCKET_OFFSET));
 	LWIP_ASSERT("newconn->callback == event_callback", newconn->callback == event_callback);
 	nsock = get_socket(newsock, getpid());
+	if (nsock == NULL) {
+		netconn_delete(newconn);
+		free_socket(nsock, 1);
```

Comment:
As 'nsock' is already NULL, "free_socket(nsock, 1)" would occurs an error.
In 'free_socket', It is using 'nsock' without checking NULL.

Reply:
Thanks. I made mistake. I wanted to free `newsock`, so fixed.